### PR TITLE
[TWEAK] Climbing into delay is now affected by your stat Vigilance

### PR DESCRIPTION
**Pull request by TorinoFermic**
## Changelog
🆑 TorinoFermic
tweak: Vigilance will affect how fast you climb onto/into disposals, ladders, tables and railings.
/🆑

### CODE REVIEW

File: code/_onclick/other_mobs.dm [hunk 1]
```diff
@@ -52,7 +52,8 @@
 
 			visible_message(SPAN_WARNING("[src] starts climbing onto \the [A]!"))
 			shadow.visible_message(SPAN_WARNING("[shadow] starts climbing onto \the [A]!"))
-			if(do_after(src, 50, helper))
+			var delay = 50
```
Comment (LethalGhost) [inline code/_onclick/other_mobs.dm]:
```suggestion
			var/delay = 50
```

Comment (LethalGhost) [reply code/_onclick/other_mobs.dm]: Please keep codestyle solid with all other files.

Comment (TorinoFermic) [reply code/_onclick/other_mobs.dm]: How I could have missed this detail, although that's my experience of Java leaking in my code.

**Comment by TorinoFermic** [comment]
Fixed, anything else to fix like changing the min of 66% for something better ?
I had put that otherwise IH will have instant climbing on everything like tables, etc.
All stats checks uses the max of STAT_LEVEL_EXPERT which is 40 stats points. And the reduction is capped to 66% of original timer.

**Pull request merged**